package.json

```diff
@@ -22,7 +22,7 @@
     "kibble:build": "npm-run-all init css:build js:build && cd output && kibble publish",
     "css:local:backup": "cp ./output/site/styles/main.scss ./output/site/styles/_main.scss",
     "css:local:variables": "concat -o ./output/site/styles/_variables.scss ./local/site/styles/_variables.scss ./node_modules/@shift72/core-template/site/styles/_variables.scss && npm run css",
-    "css:local:custom": "concat -o ./output/site/styles/main.scss ./output/site/styles/_main.scss ./local/site/styles/local.scss && npm run css:local:variables",
+    "css:local:custom": "cp -RT ./local/site/styles ./output/site/styles/local && concat -o ./output/site/styles/main.scss ./output/site/styles/_main.scss && npm run css:local:variables",
```

Comment:
I'm having trouble understanding the `concat` now. Previously it was:

```
concat
   -o ./output/site/styles/main.scss  (output file)
      ./output/site/styles/_main.scss (input 1)
      ./local/site/styles/local.scss  (input 2)
```

Now it is (?):

```
concat
  -o ./output/site/styles/main.scss  (output file)
     ./output/site/styles/_main.scss (just the one input?)
```

If that is intended:

1. Why `concat` and not just rename (or `cp`)?
2. Do we not have a `local.scss` anymore? How are things overriding?

Reply:
And I know you said "Copy scss files to output instead of concatenate them to main scss" but I'm missing something.

Reply:
We couldn't `@import` or `@use` in `local.scss` is because it was concatenated to `main.scss` before.

Now `local.scss` (and any other files in that directory) are copied instead to `./output/site/styles/local` and the `main.scss` already has an `@import 'local/local'` statement.  My thinking behind this is:
1. We can use `@import` and `@use` in `local.scss`.
2. We can name our local scss files whatever we want without worrying about clashing with the ones in core template because they are tucked into that `local` directory.

We just have to enforce a rule that `local/local` is the last import in `main.scss` to guarantee that it overrides (in the same way that concatenating to the end of the file does).

Reply:
I understand the change to allow `local.scss` to `@import`/`@use`, but I'm not sure it's answered my first question:

> Why concat and not just rename (or cp)?

For example, instead of `concat -o main.scss _main.scss` just `cp -f _main.scss main.scss`? And it begs the question if we aren't concatentaing multiple files anymore, do we even need `_main.scss` - why not just have `main.scss`?

The reason why I'm digging in on `concat` is to see if we can potentially remove a dependency.

Reply:
Ah gotcha... yeah I think we can probably ditch `concat`.  I'll keep that in mind while I sort out a js tool to run sass smarter.